vmware_aria_operations_integration_sdk/serialization.py

```diff
@@ -342,13 +342,19 @@ def __init__(
         )
 
 
-class WaitBundle:
+class WaitBundle(ResponseBundle):
     def __init__(
         self, duration: float, container_statistics: Optional[ContainerStats]
     ) -> None:
         self.duration = duration
         self.container_statistics = container_statistics
 
+    def validate(self, project: Project) -> Result:
+        return Result()
+
+    def failed(self) -> bool:
+        return False
+
```

Comment:
Because of how the 'run' method is structured, it assumes that the response for any method implements the `ResponseBundle` interface. These aren't necessarily relevant for `WaitBundle`, but it was cleaner to implement stubs here than introduce a special case in the calling code.